app/forms/provider_interface/provider_user_invitation_wizard.rb

```diff
@@ -1,17 +1,15 @@
 module ProviderInterface
   class ProviderUserInvitationWizard
     include ActiveModel::Model
-    STATE_STORE_KEY = :provider_user_invitation_wizard
+    include Wizard
 
-    attr_accessor :current_step, :current_provider_id, :first_name, :last_name, :checking_answers, :single_provider, :view_applications_only
+    attr_accessor :current_provider_id, :first_name, :last_name, :checking_answers, :single_provider, :view_applications_only
```

Comment:
I think `:checking_answers` can go here?

Reply:
I used `review_mode` in the wizard (as this is what it does in the interview form, enters the review page) but here its' called checking answers so not. We could tackle this by additional refactoring of the provider user invitation wizard but I think it's better to tackle it separately.

Reply:
It doesn't look like it would be too big to remove checking_answers. I think it's worth doing this now since the check_answers variable seems to do the opposite of what you'd expect